lavalink/src/player.rs

```diff
@@ -39,19 +39,20 @@ impl PlayerManager {
     }
 
     /// Return an immutable reference to a player by guild ID.
-    pub fn get(&self, guild_id: &GuildId) -> Option<Player> {
+    pub fn get(&self, guild_id: &GuildId) -> Option<Arc<Player>> {
         // Clippy recommends removing the `map` call, which is just wrong.
```

Comment:
Should this comment be removed too now that the clippy allow code is gone?

Reply:
Done.